lib/broadside/deploy.rb

```diff
@@ -32,33 +29,27 @@ def full
     # The `yield` calls are a little weird but this was designed with an eye towards supporting other docker
     # based systems beyond ECS.  That day hasn't come yet, but we didn't think it was worth undoing the structure.
 
-    def rollback(count = @rollback)
+    def rollback(count)
       info "Rolling back #{count} release for #{family}..."
       yield
       info 'Rollback complete.'
     end
 
-    def scale
```

Comment:
i got rid of the super class method on a whim but also because if we do this refactor, i think the superclass will have to die because if `Deploy` and `Command` areModules instead of classes they won't really handle inheritance the same.